cps_data/finalprep.py

```diff
@@ -341,8 +341,26 @@ def adjust(data, targets):
 def benefits(data, other_ben):
     """
     Distribute benefits from non-models benefit programs and create total
-    benefits variable
+    benefits variable.
+    Replaces Medicare and Medicaid values with set amounts
     """
+    # replace medicare and medicaid
```

Comment:
This code should run faster than lines 348-354:

```
medicare_cols = 'MCARE_VAL' + pd.Series((np.arange(16) + 1).astype(str))
medicaid_cols = 'MCAID_VAL' + pd.Series((np.arange(16) + 1).astype(str))

count_medicare = data[medicare_cols].astype(bool).sum(axis=1)
count_medicaid = data[medicaid_cols].astype(bool).sum(axis=1)
```

See https://drive.google.com/file/d/1Fw8rcvcERKs9llMf6dfOVAPuqwuqUUah for an example.

Reply:
Thanks for the tip!